sys/ztimer/Kconfig

```diff
@@ -5,6 +5,11 @@
 # directory for more details.
 #
 
+config HAS_ZTIMER_PERIPH_LPTIMER_CONFIG
```

Comment:
With that in mind, I would just change it to
```suggestion
config HAVE_ZTIMER_PERIPH_LPTIMER
```

Reply:
Maybe update it everywhere.  Also, are there any places where this is applied?  I would assume this PR should contain some boards or CPU families or something that would `HAVE_ZTIMER_PERIPH_LPTIMER`?

Reply:
If not, now that I look at the testing procedure again.  It may be nice to add in a small test that just does that?

With the current broken state of master we can see, if it isn't tested, and isn't automatic, it won't be done.

Reply:
@MrKevinWeiss: When to use HAVE_ and when  HAS_ ?

Reply:
Once #18780 is merged, the first ztimer_periph_lptimer can be added.

Reply:
Added your suggestion. Thank you!